### ISSUE TITLE
Module Crashes and I am now unable to open ANY module

### ISSUE THREAD

**Issue by vladu2131** [Beamdog/nwn-issues#682]
### OS

Windows

### Game version

1.88

### Describe the crash

[nwmain-crash-1730420650.dmp](https://github.com/user-attachments/files/17595390/nwmain-crash-1730420650.dmp)
[nwmain-crash-1730420650.log](https://github.com/user-attachments/files/17595391/nwmain-crash-1730420650.log)
I was recently able to load this module by testing thru toolkit but now I am not able to load this or ANY module. I had a recent save of this module that I opened using NWSERVER.exe and players joined. But I cannot join as player or DM.
Custom hak file: [Kragmoor.zip](https://github.com/user-attachments/files/17595409/Kragmoor.zip)
Hak has been included for a while with no issues.

**Comment by tinygiant98** [comment]
Did you get any messages in your server log before the crash?

**Comment by vladu2131** [comment]
Well, it looks like I somehow solved the problem. I didn't see any messages. I tried removing the hak (still crashed) then re-added it (still crashed). I then backed out a few changes I made and it seems like when I took a couple creatures out of an area it just started working again.

The main thing I have concerns with is that I was not able to log in to ANYTHING but for now it seems like I am good.

Thanks for the response!
